rktl_sim/urdf/walls.urdf

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <!-- =================================================================================== -->
-<!-- |    This document was autogenerated by xacro from /home/hsan/catkin_ws/src/rktl_3/rktl_sim/urdf/walls.urdf.xacro | -->
+<!-- |    This document was autogenerated by xacro from /home/abuynits/catkin_ws/src/rocket_league/rktl_sim/urdf/walls.urdf.xacro | -->
```

Comment:
I added the whole folder to the .git ignore ie rktl_sim/urdf. All of the files in it are autogenerated. is this what you are looking for?

Reply:
No, `xacro` files generate the `urdf` files. So we want to leave those out of the `.gitignore`. Since we want to keep some `urdf` files, just explicitly state the autogenerated ones.

Reply:
Cool!
These are the ones I added that were autogenerated
rktl_sim/urdf/walls.urdf
rktl_sim/urdf/goal_a.urdf
rktl_sim/urdf/goal_b.urdf
rktl_sim/urdf/walls.urdf